src/Parser/AggregateMoneyParser.php

```diff
@@ -22,6 +22,10 @@
      */
     public function __construct(array $parsers)
     {
+        if (empty($parsers)) {
+            throw new \InvalidArgumentException(sprintf('Initialize an empty %s is not possible', self::class));
```

Comment:
Can you change `self::class` in `get_class($this)`? We are supporting PHP 5.5. Which I do not like, but until we are dropping it, we have to take care of it.